src/lnav.cc

```diff
@@ -1886,7 +1886,7 @@ static void handle_key(int ch) {
     }
 
     switch (ch) {
-    case CEOF:
+    case EOF:
```

Comment:
CEOF is CTRL+D, see http://unix.superglobalmegacorp.com/Net2/newsrc/sys/ttydefaults.h.html

EOF is -1, so this isn't the same

Reply:
I have replaced CEOF in with CTRL('d') from readline. That macro is the same as the one in ttydefaults.h used to define CEOF and it makes the intent of the code a bit clearer, I think, as CTRL-D is spelled out in the code itself (I had a hard time googling for CEOF). As we use readline anyway, that should work everywhere.